adcc/MoSpaces.py

```diff
@@ -20,79 +20,157 @@
 ## along with adcc. If not, see <http://www.gnu.org/licenses/>.
 ##
 ## ---------------------------------------------------------------------
-from .backends import import_scf_results
-from .memory_pool import memory_pool
+import numpy as np
+
+from collections.abc import Iterable
 
 import libadcc
 
+from .backends import import_scf_results
+from .memory_pool import memory_pool
+
 __all__ = ["MoSpaces"]
 
 
+def expand_spaceargs(hfdata, **spaceargs):
```

Comment:
I guess this was moved here from the C-side?

Reply:
Never mind... I saw now that it came from `ReferenceState`...

Reply:
Yes indeed. It came from `ReferenceState`, where it was a duplicate from `C`-side for reasons I cannot remember, so good it's now only once at an understandable place.